CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.2.2
```

Comment:
If we yank 4.2.0 and 4.2.1, then we should probably list those as yanked, and just copy the text from 4.2.0 here.